### PR TITLE
Airbnb ESLint config

### PR DESCRIPTION
**Pull request by aannyyaa**

### CODE REVIEW

File: src/components/header.jsx
Comment (saifulislamdev) [inline src/components/header.jsx]: How will my changes in `header.js` and `footer.js` be transferred over? For `header.js`, it looks like it may be transferred over, but for `footer.js` it seems like you're deleting the file entirely and creating a new one.

File: src/templates/landing-page.jsx
Comment (saifulislamdev) [inline src/templates/landing-page.jsx:47]: Nice catch!

File: src/components/header.jsx
Comment (aannyyaa) [reply src/components/header.jsx]: I renamed all the files that use JSX by using the syntax `git mv oldfile.js newfile.jsx`. Following the airbnb eslint guidelines in many cases two quotation marks are changed to one. Maybe if there were changes throughout the whole file and the file was moved it says it was deleted? Not sure why it considers footer deleted but not header

File: src/templates/landing-page.jsx
Comment (aannyyaa) [reply src/templates/landing-page.jsx:47]: I wish I caught it myself but it was airbnb eslint!

File: src/components/header.jsx
Comment (saifulislamdev) [reply src/components/header.jsx]: Let's merge this after my PRs are merged.

File: src/components/footer.jsx
Comment (saifulislamdev) [inline src/components/footer.jsx:16]: I think it would be best to leave out the changes in the components until my PRs are merged. Once they're merged, you can merge dev into your branch to make sure the changes are made in the components and then create the necessary .jsx files.

**Review by saifulislamdev — APPROVED** [review]

**Pull request merged**
